test/bdd/pkg/verifiable/verifiable_steps.go

```diff
@@ -147,16 +136,21 @@ func (s *SDKSteps) addVCProof(vc *verifiable.Credential, issuer, proofType strin
 }
 
 func (s *SDKSteps) verifyCredential(holder string) error {
```

Comment:
Here we use the same approach for VC verification. `WithPublicKeyFetcher()` is required for both JWS and Linked Data Proof verification. `WithEmbeddedSignatureSuites()` is LDP-specific.